### ISSUE TITLE
blog code to update with latest libsql API

### ISSUE THREAD

**Issue by tjyang** [tursodatabase/libsql#551]
**WHAT:** Please update source code in [Microsecond-level SQL query latency with libSQL local replicas](https://blog.turso.tech/microsecond-level-sql-query-latency-with-libsql-local-replicas-5e4ae19b628b)

**WHY**:  provide the beginner a jumpstart with a working "cargo run"  environment.

**Comment by avinassh** [comment]
This example should work: https://github.com/tursodatabase/libsql/blob/main/libsql/examples/remote_sync.rs

**Comment by tjyang** [comment]
@avinassh , Yes. I was able to build and start up a sqld , using turso cli to create a test db inside sqld and use remote_sysnc.rs to connect to http://127.0.0.1:8080 to demo a remote sync to local db.
But still the tutorial is better to be updated to match current libsql repo hosted under tursodabase github org.

[R1](https://docs.google.com/document/d/14XXByLQcJkEzDUtvcWq6HKp-nnxBhS_Pvoawvwotr_0/edit) is my learning notes in gdoc.
